### DISCUSSION TITLE
Lumen and mail template after upgrade to 9

### DISCUSSION THREAD

**Discussion by paraflu** [laravel/framework#45381] [Q&A]
After upgrade to 9 all the email fail with strange error.

`[2022-12-21 11:42:16] local.ERROR: Target [Illuminate\View\ViewFinderInterface] is not instantiable while building [Illuminate\View\Factory]. (View: D:\projects\blog\resources\views\emails\test.blade.php) {"exception":"[object] (Illuminate\\View\\ViewException(code: 0): Target [Illuminate\\View\\ViewFinderInterface] is not instantiable while building [Illuminate\\View\\Factory]. (View: D:\\projects\\blog\\resources\\views\\emails\\test.blade.php) at D:\\projects\\blog\\vendor\\illuminate\\container\\Container.php:1091)
`

If I revert to 8 no problems. I've do some test, recheck the documentation, create a empty project and I've the same error.

I've tried with simple view without <x-mail::message> component and it work, so It isn't a configuration problem.

Some help?

**Comment by birdspider** [comment]
somehow (after hours trying to wrap my head around ViewFinder) I realized:

 lumen 10 and illuminate/mail 10 (and maybe 9) don't understand the newer `x-mail` syntax.

I specifically had to use the old vendor templates (i.e. the ones BEFORE this commit: https://github.com/illuminate/mail/commit/d652269320256467465bfab9f54933d9b8a1ce2f). (yes even in a fresh lumen 10 project)

Which is in contrast to what the official docs say (just copy that vendor folder). I hope this saves someone some frustrating hours.

**Reply by macropay-solutions** [reply]
Glad to see other people still using lumen.
